src/font/font_options.rs

```diff
@@ -0,0 +1,115 @@
+use glib::translate::*;
+use std::cmp::PartialEq;
+use ffi;
+
+use ffi::enums::{
+    Antialias,
+    SubpixelOrder,
+    HintStyle,
+    HintMetrics,
+};
+use ffi::cairo_font_options_t;
+
+glib_wrapper! {
+    pub struct FontOptions(Boxed<ffi::cairo_font_options_t>);
+
+    match fn {
+        copy => |ptr| ffi::cairo_font_options_copy(ptr),
+        free => |ptr| ffi::cairo_font_options_destroy(ptr),
+    }
+}
+
+impl FontOptions {
+    pub fn new() -> FontOptions {
+        let mut font_options: FontOptions = unsafe {
+            from_glib_full(ffi::cairo_font_options_create())
+        };
+        font_options.ensure_status();
+        font_options
+    }
+
+    #[doc(hidden)]
+    pub fn get_ptr(&self) -> *const cairo_font_options_t {
+        self.to_glib_none().0
+    }
+
+    #[doc(hidden)]
+    pub fn get_ptr_mut(&mut self) -> *mut cairo_font_options_t {
+        self.to_glib_none_mut().0
+    }
+
+    pub fn ensure_status(&mut self) {
+        let status = unsafe {
+            ffi::cairo_font_options_status(self.get_ptr_mut())
```

Comment:
This don't need mutable pointer.
If  `mut_override(self.to_glib_none().0)` used then even don't need change ffi part.